week-3/Homework/mandatory/1-oxygen-levels.js

```diff
@@ -7,17 +7,44 @@ Their computer detects a list of nearby planets that have Oxygen in their atmosp
 To be safe, they need to land on the first unamed planet that has Oxygen levels between 19.5% and 23.5%.
 
 Write a function that finds the oxygen level of the first safe planet - Oxygen between 19.5% and 23.5%
+
+Some string methods that might help you here are .replace() and .substring(). Let's look at a quick
+example before trying the exercise.
 */
 
-function safeLevels() {
+/* .replace() allows us to add something where we removed something*/
+let greeting = "Good Morning";
+greeting.replace('Morning', 'Evening'); // outputs Good Evening
 
-}
 
+/* .substring() allows us to remove things from strings */
+let dessert = "ice cream and pancakes";
+
+let newdessert = dessert.substring(0, 9);
+
+console.log(newdessert); // returns ice cream
+
+/* + + + + + + + */
+/* Now try the exercise */
+
+// function safeLevels(arr) {
+//   for (let i=0; i<arr.length ; i++){
+//     arr[i] = arr[i].replace("%","")
+//     if (arr[i] > 19.5 && arr[i] <23.5){
+//     return arr[i].concat("%");
+// }
+//   }}
+function safeLevels(levels) {
+  var O2 = levels.map(arr => arr.substring(0, 4))
+  var levelNum = O2.find(function(item){
+   return item > 19.5 && item < 23.5
+  });
+  return levelNum.concat("%")
+}
```

Comment:
You have a couple of mistakes here, first one is in the variable O2 a variable name should not start by capital letter. Second is that the subString that you have retrieved is a string it should be a number (google how to change a string to a number) then use find() method to check what you want and finally add the % symbol.